projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -274,6 +274,33 @@ public function get_upsell_nudge() {
 		}
 	}
 
+	/**
+	 * Adds Stats menu.
+	 */
+	public function add_stats_menu() {
```

Comment:
I wonder if we should extract this into a new function that would be used here and in the admin bar for the rest of Jetpack, instead of duplicating some of the code?

Reply:
Good idea. Done in https://github.com/Automattic/jetpack/pull/21655/commits/8efff91a82aa75c1604c1313f57d0ad336859b7e.